### PR TITLE
Extension method, dependency injection, named arguments

### PR DESCRIPTION
**Pull request by Leminode**
Added:
- An extension method in the ServiceExtensions class to reduce future code in program.cs
- Dependency injections for table management methods (TableService.cs)
- Named arguments in Index.cshtml.cs for entry deletion

Other changes:
- Code cleanup

### CODE REVIEW

File: Energy Saver/Pages/Index.cshtml.cs [hunk 1]
```diff
@@ -7,18 +8,20 @@ namespace Energy_Saver.Pages
     public class IndexModel : PageModel
     {
         private readonly ILogger<IndexModel> _logger;
+        private readonly ITableService _tableService;
```
Comment (Tauras-Rim) [inline Energy Saver/Pages/Index.cshtml.cs:11]: What does this do?

File: Energy Saver/Services/ServiceExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Energy_Saver.Services
+{
+    public static class ServiceExtensions
+    {
+        public static IServiceCollection AddTableServices(this IServiceCollection services)
+        {
+            return services.AddTransient<ITableService, TableService>();
```
Comment (Tauras-Rim) [inline Energy Saver/Services/ServiceExtensions.cs:7]: What does services.AddTransient do?

File: Energy Saver/Pages/Index.cshtml.cs (hunk 1 above)
Comment (Leminode) [reply Energy Saver/Pages/Index.cshtml.cs:11]: This allows us to access table related methods through dependency injection, reducing code and the amount of dependencies in the IndexModel class.

**Review by priolette — APPROVED** [review]
Great start to rebase our project on dependency injection

File: Energy Saver/Services/ServiceExtensions.cs (hunk 2 above)
Comment (Leminode) [reply Energy Saver/Services/ServiceExtensions.cs:7]: This method ensures that a new instance of the service is created each time it is needed, where "needed" means injected into the constructor of a dependent class (e.g. a PageModel).

**Pull request merged**